vital/video_metadata/pos_metadata_io.cxx

```diff
@@ -111,7 +112,8 @@ read_pos_file( path_t const& file_path )
   md->add( NEW_METADATA_ITEM( VITAL_META_SENSOR_PITCH_ANGLE, std::stod( tokens[ base + 1] ) ) );
   md->add( NEW_METADATA_ITEM( VITAL_META_SENSOR_ROLL_ANGLE, std::stod( tokens[base + 2] ) ) );
 
-  kwiver::vital::geo_lat_lon latlon( std::stod( tokens[ base + 3]), std::stod( tokens[ base + 4 ] ) );
+  kwiver::vital::vector_2d raw_latlon{ std::stod( tokens[ base + 4 ]), std::stod( tokens[ base + 3 ] ) };
+  kwiver::vital::geo_point latlon{ raw_latlon, SRID::lat_lon_WGS84 };
```

Comment:
similar naming issues here.  Should it be `lonlat`? and `SRID::lon_lat_WGS84`?  Or is the latter an official name?  I suppose it doesn't matter too much, but I'm sure there will be errors people will run into due to swapped lon and lat.